pkg/printers/internalversion/printer.go

```diff
@@ -256,7 +258,7 @@ func printTemplateList(list *templateapi.TemplateList, w io.Writer, opts kprinte
 	return nil
 }
 
-func printBuild(build *buildapi.Build, w io.Writer, opts kprinters.PrintOptions) error {
+func printBuild(build *buildv1.Build, w io.Writer, opts kprinters.PrintOptions) error {
```

Comment:
hm, does it make sense to have external handlers in the `internalversion` package? Might make more sense to duplicate externals into a package of their own within `oc` tree?

Reply:
agree, I don't like mixing these here as well, but I don't want to bloat this PR with that change. Will you be ok with follow up to fix this?

Reply:
Yep, followup PR is fine